odin/src/main/scala/org/clulab/odin/impl/Extractor.scala

```diff
@@ -45,29 +45,46 @@ class TokenExtractor(
     action(mentions, state)
   }
 
-  def mkMention(r: TokenPattern.Result, sent: Int, doc: Document): Mention =
-    r.groups.keys find (_ equalsIgnoreCase "trigger") match {
-      case Some(triggerKey) =>
+  def mkMention(r: TokenPattern.Result, sent: Int, doc: Document): Mention = {
+    val groupsTrigger = r.groups.keys find (_ equalsIgnoreCase "trigger")
+    val mentionsTrigger = r.mentions.keys find (_ equalsIgnoreCase "trigger")
+    (groupsTrigger, mentionsTrigger) match {
+      case (Some(groupTriggerKey), Some(mentionTriggerKey)) =>
+        // Can't have both notations
+        throw new RuntimeException("Can't specify a trigger in as both named capture and named mention")
```

Comment:
this is something i don't love -- i feel like this case could be handled more cleanly?  or is it ok to throw an exception?  if people specified 2 triggers in two ways, i guess they are likely confused about how event mentions work, right?  so maybe crashing is ok? even if so, it should prob be a diff kind of exception...?